src/controllers/routineController.js

```diff
@@ -0,0 +1,18 @@
+const { routineStartService } = require("../services");
+
+const routineStartController = async (req, res) => {
```

Comment:
- 함수 이름 명명에 전반적으로 수정이 필요할 것 같습니다.
- JS 함수는 동사로 시작하는 것이 일반적이며, 어떤 데이터를 가공하는지 명확한 것이 좋습니다.
- dao단 SQL을 보니, 운동 루틴의 정보를 SELECT하는 단순 get 함수 인 것 같아서 getExerciseRoutine가 더 명확한 것 같다고 생각됩니다.
- 혹시 제가 놓친 부분이 있다면 알려주세요!
```suggestion
const getExerciseRoutineByRoutineId = async (req, res) => {
```